pyproject.toml

```diff
@@ -1,20 +1,16 @@
-# Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
-# SPDX-License-Identifier: 0BSD
-
 [tool.poetry]
-name = "bot"
```

Comment:
This needs to be `bot`, unfortunately

Reply:
Actually I didn't realize you moved the `bot` one to the other directory (which is correct). Why is there this one at the root directory?

Reply:
just an oversight -- don't think we need the root python stuff anymore (just under `python`)